twirssi.pl

```diff
@@ -322,10 +322,10 @@ sub format_expand {
     my %args = @_;
     $args{fmt} =~ s/\$n/\@$args{nick}/g;
     if (defined $args{data} and $args{data} ne '') {
-        $args{fmt} =~ s/\${|\$}//g;
+        $args{fmt} =~ s/\$\{|\$\}//g;
```

Comment:
Dan Boger said:
> 
> In twirssi.pl:
> 
> > @@ -322,10 +322,10 @@ sub format_expand {
> >      my %args = @_;
> >      $args{fmt} =~ s/\$n/\@$args{nick}/g;
> >      if (defined $args{data} and $args{data} ne '') {
> > -        $args{fmt} =~ s/\${|\$}//g;
> > +        $args{fmt} =~ s/\$\{|\$\}//g;
> 
> I don't mind escaping the {, but I don't think they have any special meaning
> except when they contain digits?

You get a warning with newer perls if it isn't escaped. Not entirely sure 
why (haven't bothered to look, just wanted to get rid of the noise).


- Salve

-- 
#!/usr/bin/env perl
sub AUTOLOAD{$AUTOLOAD=~/.*::(\d+)/;seek(DATA,$1,0);print# Salve Joshua Nilsen
getc DATA}$"="'};&{'";@_=unpack("C*",unpack("u*",':50,$'.#    <sjn@foo.no>
'3!=0"59,6!`%%P\0!1)46%!F.Q`%01,`'."\n"));eval "&{'@_'}";  __END__ is near! :)